TicketingSystem.Common/Model/Database/Seat.cs

```diff
@@ -0,0 +1,16 @@
+﻿using TicketingSystem.Common.Model.Database.Enums;
+
+namespace TicketingSystem.Common.Model.Database
+{
+    public class Seat
+    {
+        public int SeatId { get; set; }
```

Comment:
Your DB schema missing `Section` and `Row` to identify specific seats. You'll need them later to return sections for a venue and specific section seats for events.

Also, ticket admission type is missing:
> The are two seat types - specific designated place in a row and general admission. General admission means patrons can choose to sit anywhere within a particular section (for example, the dance floor)/

Reply:
Also I do not see `Prices` table:

> **Prices –** Information about the ticket price levels (Adult, Child, VIP, etc.)

Reply:
Added tables Sections, PriceCategories, SeatTypes
Added int RowNumber to Seats